.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag (Optional, without 'v' e.g. 1.2.3)"
+
+jobs:
+  release:
+    name: Tag and release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bump version and push tag
```

Comment:
what happens if the tag already exists?

Reply:
Right, I need to test this use case. Good catch, forgot to test this one.

Reply:
The tag creation will be skipped. Per the changed release workflow in https://github.com/canonical/rocks-toolbox/pull/2/commits/b1d4989eebc6a60dc2056e0999cae459d43b0065, the release will be skipped too if the release for that tag exists. However, if a tag exists and the release for that tag doesn't, a new release will be created.